packages/store/src/usersSlice/thunks.ts

```diff
@@ -377,33 +462,42 @@ const customPermissionHandler = (accu, permission) => {
   };
 };
 
-const mapPermissionSet = (permissionSetName, names, scope, existingGroupsPermissions = {}) => {
-  const permission = Object.values(uiPermissionsById).find(permission => permission.permissionSets[scope] === permissionSetName).value;
+const mapPermissionSet = (
+  permissionSetName: PermissionSetIds,
+  names: string[],
+  scope: UiPermissionsByAreaKeys,
+  existingGroupsPermissions: CombinedPermission = {}
```

Comment:
the singular/ plural type name misalignment commented on above seem to particularly stick out here